in-app-payments/src/main/java/eu/kevin/inapppayments/paymentconfirmation/PaymentConfirmationViewModel.kt

```diff
@@ -72,6 +79,10 @@ internal class PaymentConfirmationViewModel(
         }
     }
 
+    private fun getKevinPluginLanguage(): String {
+        return Kevin.getLocale()?.language?.lowercase() ?: "en"
```

Comment:
This a bit contradicts general implementation. Kevin object has an optional language, because by default it will fall to the device language, but here you fallback to English which is not really correct.